EventGenerator/inc/PrimaryProtonGunImpl.hh

```diff
@@ -0,0 +1,140 @@
+#ifndef EventGenerator_PrimaryProtonGunImpl_hh
+#define EventGenerator_PrimaryProtonGunImpl_hh
+//
+// Generate a proton with the primary proton energy
+//
+// $Id: PrimaryProtonGunImpl.hh,v 1.17 2013/12/13 21:35:07 gandr Exp $
+// $Author: gandr $
+// $Date: 2013/12/13 21:35:07 $
```

Comment:
When you encounter these comments - just delete them.  Please do this as targets of opportunity.  They are left overs from a version control system, cvs, that we stopped using a long time ago.  With git they add no value - just noise.